entries/s/swinburne.edu.au.json

```
@@ -0,0 +1,21 @@
+{
+    "Swinburne University of Technology": {
+      "domain": "swinburne.edu.au",
+      "tfa": [
+        "sms",
```

```suggestion
        "sms",
        "call",
```

[Source](https://www.swinburne.edu.au/media/swinburneeduau/about-swinburne/docs/pdfs/mfa-setup-guide.pdf#page=7)